sympy/stats/tests/test_finite_rv.py

```diff
@@ -345,11 +386,11 @@ def test_DieDistribution():
     from sympy.abc import x
     X = DieDistribution(6)
     assert X.pdf(S(1)/2) == S.Zero
-    assert X.pdf(x).subs({x: 1}).doit() == S(1)/6
```

Comment:
what about these tests?

Reply:
I think I changed the pdf of `DieDistribution` to `Lambda` for making things work. I will try to restore the original API(though its not user facing).